### ISSUE TITLE
Contributor sees all Transactions on the Account page with the option to filter To/From.

### ISSUE THREAD

**Issue by acravenho** [poanetwork/poa-explorer#14]
Contributor should see a table below the Account Balance that shows all Transactions for an Account. There should be an option to filter To/From for that Account.

The table should include: TXHash, Block,, Age, From, To, Value, TXFee

<img width="1095" alt="screen shot 2018-04-01 at 5 37 16 pm" src="https://user-images.githubusercontent.com/17620007/38177715-6d8aa9de-35d3-11e8-8c86-a19ca642cf92.png">

**Comment by jimmay5469** [comment]
Branch: account-page-revised-transactions-list-#14

**Comment by jimmay5469** [comment]
This is blocked by #118. We would like the work on #118 to go to `master` first which will require rebase/refactor on this branch.

**Comment by tmecklem** [comment]
This work is now rebased on top of #118 that was merged into master earlier.
